modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/xeno.dm

```diff
@@ -50,8 +50,8 @@
 	icon_state = "royal"
 
 /datum/sprite_accessory/xenohead/hollywood
-	name = "Hollywood"
-	icon_state = "hollywood"
+	name = "Nethead"
+	icon_state = "net"
```

Comment:
If it's no longer called hollywood it probably shouldn't be still using the `/datum/sprite_accessory/xenohead/hollywood` typepath.